egs/reverb/asr1/local/run_wpe.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
```

Comment:
Add the shebang line. Is it based on python2 or python3? Please also specify it.

Reply:
It works on both, I will add "#!/usr/bin/env python" as the shebang

Reply:
Made this change.